deploy/examples/create-external-cluster-resources.py

```diff
@@ -934,9 +944,18 @@ def get_rbd_provisioner_caps_and_entity(self):
                     "mandatory flag not found, please set the '--k8s-cluster-name' flag"
                 )
             entity = self.get_entity(
-                entity, rbd_pool_name, alias_rbd_pool_name, k8s_cluster_name
+                entity,
+                rbd_pool_name,
+                alias_rbd_pool_name,
+                k8s_cluster_name,
+                rados_namespace,
             )
-            caps["osd"] = f"profile rbd pool={rbd_pool_name}"
+            if rados_namespace != "":
+                caps[
+                    "osd"
+                ] = f"profile rbd pool={rbd_pool_name} namespace={rados_namespace}"
```

Comment:
is this break to avoid having long lines?

Reply:
It is a black formatter